hermes-frontend/src/main/java/pl/allegro/tech/hermes/frontend/publishing/message/RequestTimeoutLock.java

```diff
@@ -0,0 +1,21 @@
+package pl.allegro.tech.hermes.frontend.publishing.message;
+
+import java.util.concurrent.locks.Lock;
+import java.util.concurrent.locks.ReentrantLock;
+
+public class RequestTimeoutLock {
```

Comment:
Not sure if this class is necessary, it just delegates everything to the ReentrantLock - a meaningful variable name should be sufficient.